dashboard/server.py

```
@@ -1045,6 +1045,86 @@ def api_system_venue_sync(request: Request):
+        funder = os.environ.get("POLY_FUNDER")
+        summary = reconcile_orders(c, reg, maker_address=funder)
+        result["steps"]["reconcile"] = {
+            "ok": True,
+            "open_orders_count": summary.open_orders_count,
+            "trades_polled": summary.trades_polled,
+            "fills_recorded": summary.fills_recorded,
+            "orders_filled": summary.orders_filled,
+            "orders_cancelled": summary.orders_cancelled,
+            "transitions": summary.transitions[:20],
+        }
+    except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)